qemu/tests/hpt_miscellaneous.py

```diff
@@ -88,6 +89,7 @@ def run(test, params, env):
         # For HPT reszing after hotplug memory
         hpt_mem = MemoryHotplugTest(test, params, env)
         hpt_mem.hotplug_memory(vm, "hpt_mem")
+        time.sleep(20)
```

Comment:
Short term: Consider reading it from a config, so that it adjusted accordingly in the course of run...
Long term: better to implement a conditional check if possible, something similar to https://github.com/autotest/tp-libvirt/blob/master/libvirt/tests/src/libvirt_vcpu_plug_unplug.py#L347